File: dest/17_staticCompanionObject.swift
class MyClass {
  func a() -> Int {
    return 1
  }
  let d = 4
  var g = 7

    static func b() -> Int {
      return 2
    }
    static let e = 5
    static var h = 8

  func c() -> Int {
    return 3
  }
  let f = 6
  var i = 9
  init() {
  }
}

func main(args: [String]) {
  let obj = MyClass()

  print(obj.a())
  print(MyClass.b())
  print(obj.c())

  print(obj.d)
  print(MyClass.e)
  print(obj.f)

  print(obj.g)
  print(MyClass.h)
  print(obj.i)
}

main([])

File: test-src/17_staticCompanionObject.kt
class MyClass {
  fun a(): Int {
    return 1
  }
  val d = 4
  var g = 7

  companion object {
    fun b(): Int {
      return 2
    }
    val e = 5
    var h = 8
  }

  fun c(): Int {
    return 3
  }
  val f = 6
  var i = 9
}

fun main(args: Array<String>) {
  val obj = MyClass()

  println(obj.a())
  println(MyClass.b())
  println(obj.c())

  println(obj.d)
  println(MyClass.e)
  println(obj.f)

  println(obj.g)
  println(MyClass.h)
  println(obj.i)
}